tests/test_native_agent_identity.py

```
@@ -0,0 +1,325 @@
+        grants=grants,
+        data_dir=data_dir,
+        signing_key_pem=keypair[0],
+        user_id=user_id,
+    )
+
+
+@pytest.mark.asyncio
+class TestNativeAgentIdentity:
+    async def test_mints_an_owned_active_identity_with_a_bus_handle(self, tmp_path):
+        stores = await _stores(tmp_path)
+        registry, grants, data_dir, _ = stores
```

Ruff (0.16.1)

[warning] Unpacked variable `registry` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)